tests/func/test_import.py

```diff
@@ -27,6 +29,40 @@ def test_import(git, dvc_repo, erepo):
     assert git.git.check_ignore(dst)
 
 
+def test_import_git_file(git, dvc_repo, erepo):
```

Comment:
Ok, but tests in `test_import.py` require two repositories: One "remote" (one that we import from) git repository and one local (where we import to) DVC repository. As far as I can tell, tmp_dir allows you to only init repositories in tmp_dir (rather than in subdirectories). So should I create both the "local" and "remote" repositories in the same directory (tmp_dir), use some of the old fixtures (and if so, which ones), or do something else?

Reply:
There is a `erepo_dir` fixture for that.

Reply:
Great, thanks. Unfortunately, it looks like I cannot mix `tmp_dir`, `erepo_dir` and `git` fixtures in a single test (when I do, the test fails with `dvc.stage.StagePathOutsideError: stage working or file path '/tmp/dvc-test.22662.asbntg5p.deK6FmqgHrp8evkQJ7q7sL' is outside of dvc repo` in `dvc/stage.py:396`), and the `git.git` is where the method `check_ignore` exists. I couldn't locate this method in the source code or other python modules.

Any suggestions how I might either:
* use the `git` fixture alongside `tmp_dir` and `erepo_dir`, or
* use the `check_ignore` method (or equivalent)?

Reply:
There is an `scm` fixture for that. The plan is to replace `repo_dir, git, dvc_repo, erepo` with `tmp_dir, scm, dvc, erepo_dir` you can't mix new with old ones. This is a kind of transition mess)

Reply:
@Baranowski please excuse us for this inconvenience, we are in the middle of refactoring our tests, so some friction is expected 🙁

Reply:
No worries. For the record, it appears that ordering of the new fixtures is important. I get an error for
```
def test_import_non_cached(tmp_dir, erepo_dir, dvc, scm):
```

but not for:
```
def test_import_non_cached(erepo_dir, tmp_dir, dvc, scm):
```

I'm flagging it only here since I'm not sure whether this is unexpected and therefore worthy of an issue.

Reply:
The order issue is probably because of leaking chdir in erepo_dir. This is already fixed, you need to rebase to latest master.